### DISCUSSION TITLE
1.3.7 (5302) Some files are not compressed properly.

### DISCUSSION THREAD

**Discussion by newsnetbbs** [aonez/Keka#1413] [General]
I found out today by chance while using it. I compressed the shortcut file and it's compressed into a plain text file.

![스크린샷 2024-03-15 오전 11 53 25](https://github.com/aonez/Keka/assets/94516173/cb19cd76-e284-46f2-b7b1-0b29a69535ae)

![스크린샷 2024-03-15 오전 11 54 19](https://github.com/aonez/Keka/assets/94516173/6dfbf9fd-b7e0-4b46-9424-286dfa6c4f1a)

The reliability of the compression is questioned. 
I suspect other special files are not compressible. However, the compression itself supported by MacOS was able to compress accurately. I hope it's resolved soon.

**Comment by aonez** [comment]
Hi @newsnetbbs,

This is a known issue. Currently Keka does not preserve metadata such as color labels or aliases. In your case you're using an alias, which is only compatible with macOS. To preserve that file with Keka you can use the TAR format and be sure to disable the "Exclude Mac resource forks" option.

A better and more compatible option, though maybe not that quick, is to use symlinks instead of aliases that are compatible in most Unix and Linux systems. This alias can be compressed with any format.

```sh
ln -s <YOUR_FILE_PATH> <YOUR_FILE_PATH_LINK_NAME>
```

This issue is being handled here: https://github.com/aonez/Keka/issues/90

**Reply by newsnetbbs** [reply]
Tank U! Bro!

When compressed with TAR, it compresses normally. But the other ZIPs are fine, but the problem is only with Keka.

**Reply by gingerbeardman** [reply]
macOS built-in Archive Utility also has the problem. I'd say it's a macOS thing rather than specific apps.
